src/electron/platform/android/setup/steps/prompt-grant-permissions.ts

```diff
@@ -3,13 +3,9 @@
 
 import { AndroidSetupStepConfig } from 'electron/platform/android/setup/android-setup-steps-configs';
 
-export const promptGrantPermissions: AndroidSetupStepConfig = deps => ({
+export const promptGrantPermissions: AndroidSetupStepConfig = stepTransition => ({
     actions: {
-        cancel: () => {
-            deps.stepTransition('prompt-choose-device');
-        },
-        next: () => {
-            deps.stepTransition('detect-permissions');
-        },
+        cancel: () => stepTransition('prompt-choose-device'),
```

Comment:
You've added cancel and next, but kept the original versions (using deps.stepTransition) on lines 9 and 12. Is that right?